yfinance/base.py

```diff
@@ -21,34 +21,32 @@
 
 from __future__ import print_function
 
-import warnings
-import time as _time
 import datetime as _datetime
-import dateutil as _dateutil
+import json as _json
+import logging
+import time as _time
+import warnings
 from typing import Optional
+from urllib.parse import quote as urlencode
 
-import pandas as _pd
+import dateutil as _dateutil
 import numpy as _np
+import pandas as _pd
 import pandas as pd
```

Comment:
Can one of these be removed?

Reply:
Totally. I avoided messing with most non-PEP 8 related stuff, save those I saw were already adopted, such as converting old .format to f-string. 

I don't fully understand why using `_pd` (or some of the other underscored imports in other parts of the codebase). I can go ahead and remove it if that's okay.

Reply:
Feels like it should be a PEP 8 violation, basically a duplicate import.

Reply:
It isn't a PEP 8 violation, technically. 

The underscore usage that is seen here is weird to me (along with a few other things I've been seeing on this codebase)... usually, _s are indicative of private modules...however, it doesn't make sense since pandas is also being imported/used within the same module with open usage (and used outside of the module as well)

Anyways, if that's okay, I'd simply remove the _pd (and replace all its usage for regular pd).

Reply:
Underscores were here before me, and you care more about style than me, so do what you think best.